gtest-parallel

```diff
@@ -186,6 +190,14 @@ class FilterFormat:
     self.print_tests("INTERRUPTED TESTS", interruptions)
     self.out.flush_transient_output()
 
+    if not self.passed:
+      return
+
```

Comment:
1. You can save a bunch of lines if you invert this to "if self.passed:".
2. Why don't you want an empty zip file? Won't that just require more logic in whatever script is calling us?